www/chromedriver/Portfile

```diff
@@ -25,18 +25,77 @@ dist_subdir         ${name}/${version}
 
 use_zip             yes
 
-checksums           rmd160  0be0c41da777153ea7859be8dd0fb33b0d85ceb4 \
-                    sha256  f04501f36af92ab869111137a66c84612e69ec729caa29b25fa4c305614cfe37 \
-                    size    7488025
-
 extract.mkdir       yes
 
 use_configure       no
 
 build {}
 
 destroot {
-    xinstall -m 755 ${worksrcpath}/chromedriver ${destroot}${prefix}/bin/
+    xinstall -m 755 \
+        ${worksrcpath}/chromedriver \
+        ${destroot}${prefix}/bin/${subport}
+}
+
+# most recent versions via
+# https://sites.google.com/a/chromium.org/chromedriver/downloads
+set latest_major_version \
+                    [lindex [split ${version} .] 0]
+
+checksums           rmd160  f977e91ba9599e51cf46425fc570c2dc1460852a \
+                    sha256  e1e884c4ddb41711defedd4e5c65b512eb2f8a20d084ef3c4cafbc824481b770 \
+                    size    7018740
+
+if {${subport} eq ${name}} {
+    conflicts       ${name}-${latest_major_version}
+} elseif {${subport} eq ""} {
```

Comment:
I'm wondering whether this `elseif` is necessary.

Reply:
I modified a copy-and-paste from the python portgroup for the subports. I’m not sure when and if a blank subport appears, but that case is in other code. I’ll defer to you if it’s necessary here or not.

Reply:
Thanks @essandess. I would say `${subport}` should never be empty in a python module, but I'd better ask somebody more proficient at this kind of ports.

@reneeotten: I'm asking for your opinion here, if you don't mind.

Reply:
@essandess, I've tested your `Portfile` locally and I don't see reasons for that `elseif` condition. Please, remove it, squash the two commits, push force and I'll merge this. Thanks.

Reply:
Thanks done.